flink-table/flink-table-api-java-bridge/src/test/java/org/apache/flink/table/factories/DataGenTableSourceFactoryTest.java

```diff
@@ -347,6 +352,31 @@ void testVariableLengthDataType() throws Exception {
                 "Custom length '21' for variable-length type (VARCHAR/STRING/VARBINARY/BYTES) field 'f3' should be shorter than '20' defined in the schema.");
     }
 
+    @Test
+    void testLengthForCollectionType() throws Exception {
+        DescriptorProperties descriptor = new DescriptorProperties();
```

Comment:
Could we have a test that tests the default is working a well, when the length is not explicitly set.